README.md

```diff
@@ -249,8 +254,8 @@ function ConfigRegistry(config){
 util.inherits(ConfigRegistry, DefaultRegistry);
 
 ConfigRegistry.prototype.set = function set(name, fn) {
-  var task = this._tasks[name] = fn.bind(this.config);
-  return task;
+  fn = fn.bind(this.config);
+  return DefaultRegistry.prototype.set.call(this, name, fn) || fn;
```

Comment:
the original body of this function did the same thing but with implicit knowledge of the DefaultRegistry.  I really don't like the way this looks, so maybe it would be better to add comments explaining the situation in the original function body?